projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -83,7 +83,7 @@ export class NavigationListComponent {
   }
 
   public navigate(item: NavItemLinkConfig): void {
-    this.navigationService.navigateWithinApp(item.matchPaths[0], { relativeTo: this.activatedRoute });
+    this.navigationService.navigateWithinApp(item.matchPaths[0], this.activatedRoute);
```

Comment:
We could probably use ht-link for the main items too right? That way they get the real href behavior

Reply:
Yep. Didn't want to club it with this PR. @arjunlalb is also going to help with this.